lib/src/contentsteering/callContentSteeringServer.ts

```diff
@@ -0,0 +1,49 @@
+interface ContentSteeringServerResponse {
+	status: number;
+	data: JSON;
+	headers: Record<string, string>;
+	url: string;
+}
+
+export async function callContentSteeringServer(
+	serverURI: string | null,
+	onSuccess: (response: ContentSteeringServerResponse) => void,
+	onError: (response: ContentSteeringServerResponse) => void,
+	onTimeout: (serverURI: string) => void,
+	timeout: number = 5000
```

Comment:
I would add a comment to this value specifying it's milliseconds. Or change the name of the variable to `timeoutMilliseconds` or something like that, just for clarity and avoiding confusion.
If want to go an extra step, even make it a constant at the top.

Reply:
Btw, i've assumed this is in milliseconds...